drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/view_configuration.rb

```diff
@@ -24,7 +24,7 @@ def report_path_array
       end
 
       def default_project_type_codes
-        [:ph, :oph, :rrh, :psh]
+        GrdaWarehouse::Hud::Project::PERFORMANCE_REPORTING.keys
```

Comment:
This is probably fine, but we'll need to adjust when we hit the 2024 updates.  Might be worth a TODO